exla/README.md

```diff
@@ -44,6 +44,19 @@ EXLA relies on the [XLA](https://github.com/elixir-nx/xla) package to provide th
     * Error message: "this file was generated by an older version of protoc which is incompatible with your Protocol Buffer headers".
     * If you have `protoc` installed on your machine, it may conflict with the `protoc` precompiled inside XLA. Uninstall, unlink, or remove `protoc` from your path to continue.
 
+### Usage with Nerves
+
+To get EXLA to work on your target Nerves device, you need to add `build_embedded: true` to your app's config.
+
+For example, if you're building an app called `:hello_nx`, you set:
+
+```elixir
+config :hello_nx, build_embedded: true
```

Comment:
Why do we need build_embedded?

Reply:
This is read by EXLA's (and Torchx's) mix.exs and is passed to the Makefile so that the dynamic libs are copied to priv instead of linked, which is necessary for Nerves to copy the data to the target image.

This also tells nerves to use a cross-compiler instead of a regular compiler